R/eval_forecasts.R

```diff
@@ -1,114 +1,111 @@
 #'   \item {Log Score} Smaller is better. Only for continuous forecasts.
-#'   See \code{\link{logs}} for more information.
+#'   See [logs()] for more information.
 #' }
 #'
 #' @param data A data.frame or data.table with the predictions and observations.
 #' Note: it is easiest to have a look at the example files provided in the
 #' package and in the examples below.
 #' The following columns need to be present:
 #' \itemize{
-#'   \item \code{true_value} - the true observed values
-#'   \item \code{prediction} - predictions or predictive samples for one
+#'   \item `true_value` - the true observed values
+#'   \item `prediction` - predictions or predictive samples for one
 #'   true value. (You only don't need to provide a prediction column if
 #'   you want to score quantile forecasts in a wide range format.)}
-#' For integer and continuous forecasts a \code{sample} column is needed:
+#' For integer and continuous forecasts a `sample` column is needed:
 #' \itemize{
-#'   \item \code{sample} - an index to identify the predictive samples in the
+#'   \item `sample` - an index to identify the predictive samples in the
 #'   prediction column generated by one model for one true value. Only
 #'   necessary for continuous and integer forecasts, not for
 #'   binary predictions.}
 #' For quantile forecasts the data can be provided in variety of formats. You
 #' can either use a range-based format or a quantile-based format. (You can
-#' convert between formats using \code{\link{quantile_to_range_long}},
-#' \code{\link{range_long_to_quantile}},
-#' \code{\link{sample_to_range_long}},
-#' \code{\link{sample_to_quantile}})
+#' convert between formats using [quantile_to_range_long()],
+#' [range_long_to_quantile()],
+#' [sample_to_range_long()],
+#' [sample_to_quantile()])
 #' For a quantile-format forecast you should provide:
-#' \itemize{
-#'   \item {prediction} - prediction to the corresponding quantile
-#'   \item {quantile} - quantile to which the prediction corresponds}
+#'   - `prediction`: prediction to the corresponding quantile
+#'   - `quantile`: quantile to which the prediction corresponds
 #' For a range format (long) forecast you need
-#' \itemize{
-#'   \item \code{prediction} the quantile forecasts
-#'   \item \code{boundary} values should be either "lower" or "upper", depending
+#'   - `prediction`: the quantile forecasts
+#'   - `boundary`: values should be either "lower" or "upper", depending
 #'   on whether the prediction is for the lower or upper bound of a given range
-#'   \item {range} the range for which a forecast was made. For a 50\% interval
-#'   the range should be 50. The forecast for the 25\% quantile should have
-#'   the value in the \code{prediction} column, the value of \code{range}
-#'   should be 50 and the value of \code{boundary} should be "lower".
-#'   If you want to score the median (i.e. \code{range = 0}), you still
+#'   - `range` the range for which a forecast was made. For a 50%% interval
```

Comment:
is %% correct markdown here?

Reply:
Hmm, I seem to remember that `%%` is the correct way to escape `%` in LaTeX rather than `\%` but I cannot find any source for this so I might have invented it :shrug: :sweat_smile:. 

Anyways, it seems that `%` doesn't need to be escape in markdown and that both `\%` and `%%` produce extra characters:

``` r
testfile <- tempfile(fileext = ".md")

write(
r"(
10%
20\%
30%%
)",
testfile
)

knitr::knit(testfile, output = stdout())
#> processing file: /tmp/RtmpPqcVnl/file301a18fe1273.md
#> 
#> 10%
#> 20\%
#> 30%%
#> A connection with                            
#> description "output"        
#> class       "textConnection"
#> mode        "wr"            
#> text        "text"          
#> opened      "opened"        
#> can read    "no"            
#> can write   "yes"
```

<sup>Created on 2021-08-04 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0.9000)</sup>

Thanks for making me check!